Tests/ApexyLoaderTests/ContentLoaderTests.swift

```diff
@@ -0,0 +1,93 @@
+@testable import ApexyLoader
+import XCTest
+
+final class ContentLoaderTests: XCTestCase {
+
+    private var contentLoader: ContentLoader<Int>!
+    private var numberOfChanges = 0
+    private var observation: LoaderObservation!
+
+    override func setUp() {
+        super.setUp()
+        
+        numberOfChanges = 0
+        contentLoader = ContentLoader<Int>()
+        observation = contentLoader.observe { [weak self] in
+            self?.numberOfChanges += 1
+        }
+
+        XCTAssertTrue(
+            contentLoader.observations.isEmpty,
+            "Нет наблюдений за другими загрузчиками")
```

Comment:
We need English texts here

Reply:
I've translated all the messages in the tests.